docker-for-windows/edge-release-notes.md

```diff
@@ -11,6 +11,51 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.3.1.0
+
+Add release date
+
+Add download URL
+
+### Upgrades
+
+- [Docker Compose 1.26.0-rc4](https://github.com/docker/compose/releases/tag/1.26.0-rc4)
+- Upgrade to Qemu 4.2.0, add Risc-V support
+
+### Bug fixes and minor changes
+
+**Hyper-V**
+
+- Create drive symlinks in VM at startup to avoid breaking setup that was using them. Fixes [docker/for-win#6628](https://github.com/docker/for-win/issues/6628).
+- Implement `fallocate` for shared filesystems. See [docker/for-win#6658](https://github.com/docker/for-win/issues/6658#issuecomment-627736820).
+
+**WSL 2**
+
+- Configure CLI to use Docker Desktop credential store.
+- Adds a restart button to the popup prompting the user to install the Linux kernel.
+- More reliable bootstrap, do not rely on `wslpath` for path translation, retry mounts on errors at startup.
+
+**Dashboard**
+
+- Fixed containers logs which were sometimes truncated. Fixes [docker/for-win#5954](https://github.com/docker/for-win/issues/5954)
+- Fixed `open with vs code` button for compose app deployed from a WSL 2 Linux Workspace.
+
+**Other fixes**
```

Comment:
What happened to the bug with the double slashes in Unix path names?

Also the file ownership.

Also the :z mounts.

All of these were on the last Edge weren't they?

Reply:
`:z` is not finished and will be in next edge (PR is open in grpcfuse).
And // was due to drive symlinks missing, we only mention the main ticket (docker/for-win#6628)

Reply:
`//var/lib` is working but `//c/Windows` no

Reply:
File ownership was not ready on monday but it has been merged since. We need to add it in notes and changelogs (see https://github.com/docker/pinata/pull/13945)